lib/reverse_sentence.rb

```diff
@@ -1,4 +1,48 @@
 # A method to reverse the words in a sentence, in place.
+# Time Complexity: O(3n) * O(n) + O(n) = O(n^2) + O(n) = O(n^2)
```

Comment:
Just like you fixed the time complexity in [reverse_words assignment](https://github.com/Ada-C10/reverse_words/pull/2/commits/0344ae5202e376f5feb2299381eef11bf6c9c0e8), this will need to be fixed accordingly.